docs/network-flow-visibility.md

```diff
@@ -610,3 +612,48 @@ and other Theia features, please refer to the
 **Starting with Antrea v1.7, support for the ELK Flow Collector has been removed.**
 Please consider using the [Grafana Flow Collector](#grafana-flow-collector-migrated)
 instead, which is actively maintained.
+
+## L7 Visibility
+
+In Addition to Layer 4 network visibility, Antrea adds Layer 7 network flow visibility
+enabling users to collect and visualize application-layer flow data in Theia.
+
+### Usage
+
+To achieve L7 (Layer 7) network flow visibility, the `L7Visibility` and `TrafficControl`
```

Comment:
Is enabling the `TrafficControl` FeatureGate still a requirement with the latest design? I'm asking because I think it's not required for `L7NetworkPolicy`, and the implementation is similar.

Reply:
Yes we would need TrafficControl as TrafficControlMarkFlows relies on TrafficControl feature gate.

Reply:
I still don't think that `L7Visibility` should require `TrafficControl` to be enabled, from a user perspective. Maybe the code should be updated so that the `TrafficControl` OVS table is present / initialized as long as either `L7Visibility` or `TrafficControl` is enabled. However, the `TrafficControl` API (controller) should of course not be available unless the `TrafficControl` feature gate is enabled.

Would like to hear @tnqn's thoughts on this (after he is back from vacation).

Reply:
I have changed the code where we append TrafficControlTables and trafficControlCommonFlows if any of l7Visibility or TrafficControl feature gate is enabled.
This helps us not rely on TrafficControl feature gate.